src/app/ReadHandler.h

```diff
@@ -142,6 +142,7 @@ class ReadHandler : public Messaging::ExchangeDelegate
     void MoveToState(const HandlerState aTargetState);
 
     const char * GetStateStr() const;
+    bool MergeOverlappedAttributePath(ClusterInfo & aTargetAttributePath);
```

Comment:
please add a comment of what this does. From the name, this "merges an overlapped attribute path" and the argument name implies a 'target attribute path'. This reads to me as 'target for merge' which seems inaccurate (the method merges inside mpAttributeClusterInfoList it seems).

Question needing answered: 
   - merge where (existing internal list)
   - what does the input argument mean

Reply:
Yep, update the comments, it used to merges aAttributePath inside the existing internal mpAttributeClusterInfoList